### PR TITLE
[ja] Translate tasks/administer-cluster/migrating-from-dockershim into Japanese

### LINKED ISSUE #31249: [ja] Translate tasks/administer-cluster/migrating-from-dockershim into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/administer-cluster/migrating-from-dockershim](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #31249

### CODE REVIEW

**Comment by ptux** [comment]
The changes already got reviewed.

https://github.com/kubernetes/website/pull/31250

This PR is to solve the conflict.

File: content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!-- overview -->
+
+dockershimから他のコンテナランタイムに移行する際に知っておくべき情報を紹介します。
+
+Kubernetes 1.20で[dockershim deprecation](blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)が発表されてから、様々なワークロードやKubernetesインストールにどう影響するのかという質問が寄せられています。
+
+この問題をよりよく理解するために、[Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/)ブログが役に立つでしょう。
+
+dockershimから代替のコンテナランタイムに移行することが推奨されます。
+[コンテナランタイム](/docs/setup/production-environment/container-runtimes/)のセクションをチェックして、どのような選択肢があるかを確認してください。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md:16]:
```suggestion
[コンテナランタイム](/ja/docs/setup/production-environment/container-runtimes/)のセクションをチェックして、どのような選択肢があるかを確認してください。
```

The page has already been translated

**Comment by nasa9084** [comment]
in this case, you can just do rebase and force-push because you're working on your own fork

/assign
/approve

**Comment by ptux** [comment]
@inductor

Could you give a lgtm if the change is OK to you.

**Review by riita10069 — APPROVED** [review]

**Pull request merged**